Pod/Classes/Artsy+UILabels.m

```diff
@@ -65,7 +57,10 @@ - (void)awakeFromNib
 - (void)setup
 {
     self.numberOfLines = 0;
-    self.preferredMaxLayoutWidth = (UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPad) ? 728 : 280;
+
+    CGFloat commonMargin = (UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPad) ? 40 : 20;
+    self.preferredMaxLayoutWidth = CGRectGetWidth([[UIScreen mainScreen] applicationFrame]) - (commonMargin * 2);
```

Comment:
That's tricky since it can't go in `setup` since it doesn't have a superview yet. Maybe, `willMoveToSuperview:`?